core/templates/pages/learner-dashboard-page/learner-dashboard-icons.component.ts

```diff
@@ -70,6 +70,13 @@ export class LearnerDashboardIconsComponent implements OnInit {
       );
   }
 
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.activityActive !== undefined) {
+      this.setHoverState(changes.activityActive.currentValue);
+      this.canActivityBeAddedToLearnerPlaylist(this.activityId);
```

Comment:
Does this line actually do anything?

Reply:
Removed `this.canActivityBeAddedToLearnerPlaylist(this.activityId)`